ding/policy/bcq.py

```diff
@@ -264,26 +330,44 @@ def _forward_eval(self, data: dict) -> Dict[str, Any]:
         return {i: d for i, d in zip(data_id, output)}
 
     def _init_collect(self) -> None:
+        """
+        Overview:
+            Initialize the collect mode of policy, including related attributes and modules. For A2C, it contains the \
```

Comment:
Not A2C. Maybe there are more similar mistakes, please check it.

Reply:
fixed now